src/Compute/Compute/help/New-AzRestorePoint.md

```diff
@@ -25,18 +25,33 @@ Create's a new Restore Point
 
 ### Example 1
 ```powershell
-New-AzRestorePoint -ResourceGroupName <String> -RestorePointCollectionName <String> -Name <String> [-DisksToExclude] <String[]>
+New-AzRestorePoint -ResourceGroupName "myRG" -RestorePointCollectionName "myCollection" -Name "myRestorePoint" -ConsistencyMode "CrashConsistent"
 ```
 
 Creates a new Restore Point
 
 ## PARAMETERS
 
+### -ConsistencyMode
+ConsistencyMode of the RestorePoint. Can be specified in the input while creating a restore point. For now, only CrashConsistent is accepted as a valid input. Please refer to https://aka.ms/RestorePoints for more details.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
 ### -DefaultProfile
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Type: IAzureContextContainer
```

Comment:
When refreshing documents, make sure to add `-UseFullTypeName` to `Update-MarkdownHelpModule` so the type names are full (with namespace).